docs/src/pages/demos/pickers/MaterialUIPickers.tsx

```diff
@@ -1,60 +1,45 @@
 import 'date-fns';
 import React from 'react';
-import PropTypes from 'prop-types';
 import Grid from '@material-ui/core/Grid';
-import { createStyles, withStyles, WithStyles } from '@material-ui/core/styles';
+import { makeStyles, createStyles } from '@material-ui/core/styles';
 import DateFnsUtils from '@date-io/date-fns';
 import { MuiPickersUtilsProvider, TimePicker, DatePicker } from 'material-ui-pickers';
 
-const styles = createStyles({
-  grid: {
-    width: '60%',
-  },
-});
-
-export type Props = WithStyles<typeof styles>;
-
-export interface State {
-  selectedDate: Date;
-}
-
-class MaterialUIPickers extends React.Component<Props, State> {
-  state = {
-    // The first commit of Material-UI
-    selectedDate: new Date('2014-08-18T21:11:54'),
-  };
-
-  handleDateChange = (date: Date) => {
-    this.setState({ selectedDate: date });
-  };
-
-  render() {
-    const { classes } = this.props;
-    const { selectedDate } = this.state;
-
-    return (
-      <MuiPickersUtilsProvider utils={DateFnsUtils}>
-        <Grid container className={classes.grid} justify="space-around">
-          <DatePicker
-            margin="normal"
-            label="Date picker"
-            value={selectedDate}
-            onChange={this.handleDateChange}
-          />
-          <TimePicker
-            margin="normal"
-            label="Time picker"
-            value={selectedDate}
-            onChange={this.handleDateChange}
-          />
-        </Grid>
-      </MuiPickersUtilsProvider>
-    );
+const useStyles = makeStyles(
+  createStyles({
```

Comment:
I don't think we need `createStyles` anymore if this is the immediate argument of `makeStyles`. Does this still pass without `createStyles`?

Reply:
I didn't get any autocomplete suggestions without it, in my own project it works without `createStyles`

Reply:
With the most recent version? Autocomplete should be available since #15366

Reply:
Yes, however, with the changes from https://github.com/mui-org/material-ui/pull/15402 it works without `createStyles`

Reply:
This is good to hear. I think that we should delay the removal of `createStyles()`. Waiting on #15402 to be merged would be safer.